File: packages/java/parser-jvm-plugin-backbone/src/test/java/com/vaadin/hilla/parser/plugins/backbone/multiendpoints/MultiEndpointsBarEndpoint.java
package com.vaadin.hilla.parser.plugins.backbone.multiendpoints;

import com.vaadin.hilla.Endpoint;

@Endpoint
public class MultiEndpointsBarEndpoint {
    public String getBar() {
        return "bar";
    }

    public MultiEndpointsSharedModel getShared() {
        return new MultiEndpointsSharedModel();
    }
}

File: packages/java/parser-jvm-plugin-backbone/src/test/resources/com/vaadin/hilla/parser/plugins/backbone/multiendpoints/expected/MultiEndpointsBarEndpoint.ts
import { EndpointRequestInit as EndpointRequestInit_1 } from "@vaadin/hilla-frontend";
import type MultiEndpointsSharedModel_1 from "./com/vaadin/hilla/parser/plugins/backbone/multiendpoints/MultiEndpointsSharedModel.js";
import client_1 from "./connect-client.default.js";
async function getBar_1(init?: EndpointRequestInit_1): Promise<string | undefined> { return client_1.call("MultiEndpointsBarEndpoint", "getBar", {}, init); }
async function getShared_1(init?: EndpointRequestInit_1): Promise<MultiEndpointsSharedModel_1 | undefined> { return client_1.call("MultiEndpointsBarEndpoint", "getShared", {}, init); }
export { getBar_1 as getBar, getShared_1 as getShared };

